src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Security/AuthorizationHandlers/ApproveSpecificMethodologyAuthorizationHandlerTests.cs

```diff
@@ -84,39 +113,142 @@ public async Task UserWithCorrectClaimCanApproveNonPubliclyAccessibleApprovedMet
+
+                    userReleaseRoleRepository
+                        .Setup(s => s.UserHasAnyOfRolesOnLatestRelease(UserId, OwningPublication.Id, ApproverRoles))
+                        .ReturnsAsync(ApproverRoles.Contains(releaseRole));
+
+                    var user = CreateClaimsPrincipal(UserId);
+
+                    var authContext =
+                        CreateAuthorizationHandlerContext<ApproveSpecificMethodologyRequirement, Methodology>(user,
+                            Methodology);
+
+                    await handler.HandleAsync(authContext);
+                    VerifyAllMocks(methodologyRepository, userReleaseRoleRepository);
+
+                    // If the user has the Approver role on the latest Release of the owning Publication of this
+                    // Methodology they are allowed to approve it
+                    var expectedToPass = releaseRole == Approver;
 
-                    // Only the ApproveAllMethodologies claim should allow approving a Methodology
-                    var expectedToPass = claim == SecurityClaimTypes.ApproveAllMethodologies;
                     Assert.Equal(expectedToPass, authContext.HasSucceeded);
                 });
             }
+
+            [Fact]
+            public async Task UsersWithNoRolesOnOwningPublicationsLatestReleaseCannotApprove()
+            {
+                var (
+                    handler,
+                    methodologyRepository,
+                    userReleaseRoleRepository
+                    ) = CreateHandlerAndDependencies();
+
+                methodologyRepository.Setup(mock => mock.IsPubliclyAccessible(Methodology.Id))
+                    .ReturnsAsync(false);
+
+                methodologyRepository
+                    .Setup(s => s.GetOwningPublicationByMethodologyParent(Methodology.MethodologyParentId))
+                    .ReturnsAsync(OwningPublication);
+
+                userReleaseRoleRepository
+                    .Setup(s => s.UserHasAnyOfRolesOnLatestRelease(UserId, OwningPublication.Id, ApproverRoles))
+                    .ReturnsAsync(false);
+
+                var user = CreateClaimsPrincipal(UserId);
+
+                var authContext =
+                    CreateAuthorizationHandlerContext<ApproveSpecificMethodologyRequirement, Methodology>
+                        (user, Methodology);
+
+                await handler.HandleAsync(authContext);
+                VerifyAllMocks(methodologyRepository, userReleaseRoleRepository);
+
+                // A user with no role on the owning Publication of this Methodology is not allowed to approve it
+                Assert.False(authContext.HasSucceeded);
+            }
         }
 
-        private static (ApproveSpecificMethodologyAuthorizationHandler, Mock<IMethodologyRepository>)
+        private static
+            (ApproveSpecificMethodologyAuthorizationHandler,
+            Mock<IMethodologyRepository>,
+            Mock<IUserReleaseRoleRepository>
+            )
```

Comment:
I think we should do this but I haven't done it on this PR for now as it would involve change them all to be consistent. I  prefer the Build/Setup approach that we have in services too. We can also discuss this change with @duncan-at-hiveit next week.

Reply:
Cool, no problem. Understand there might be a fair bit to refactor here!